### PR TITLE
Feature: Add enzyme oracle to composed price oracles

### LINKED ISSUE #384: Support MLN Treasury Vault token
https://etherscan.io/token/0x45c45799bcf6c7eb2df0da1240be04ce1d18cc69#readProxyContract

### PR DESCRIPTION
**Pull request by mikheevm**
### What was wrong?

- Closes #384 
- Also sets the default env file for manage.py and increases sleep time in `run_tests.py.` Let me know if you want it as a separate PR

### How was it fixed?

- Adds EnzymeOracle from gnosis-py 3.4.0 to composed price oracles.

@gnosis/safe-services

Waiting for https://github.com/gnosis/gnosis-py/pull/87 to be merged

### CODE REVIEW

File: safe_transaction_service/history/tests/utils.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def just_test_if_mainnet_node() -> str:
 
     if not mainnet_node_url:
         pytest.skip("Mainnet node not defined, cannot test oracles", allow_module_level=True)
-    elif requests.get(mainnet_node_url).status_code == 404:
+    elif requests.post(mainnet_node_url).status_code == 404:
```
Comment (mikheevm) [inline safe_transaction_service/history/tests/utils.py:14]: The infura nodes do not respond to get requests. Also, this is how it's done in gnosis_py except for the fact that gnosis-py sends eth_blockNumber request

Comment (Uxio0) [reply safe_transaction_service/history/tests/utils.py]: Good catch!

File: safe_transaction_service/tokens/services/price_service.py [hunk 2]
```diff
@@ -210,7 +211,7 @@ def get_underlying_tokens(self, token_address: ChecksumAddress) -> Optional[List
             try:
                 return oracle.get_underlying_tokens(token_address)
             except OracleException:
-                logger.info('Cannot get eth value for token-address=%s from %s', token_address,
+                logger.info('Cannot get an underlying token for token-address=%s from %s', token_address,
```
Comment (Uxio0) [inline safe_transaction_service/tokens/services/price_service.py:214]: Nice

File: manage.py [hunk 3]
```diff
@@ -5,6 +5,7 @@
 
 if __name__ == '__main__':
     os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'config.settings.local')
+    os.environ.setdefault('DJANGO_DOT_ENV_FILE', '.env')
```
Comment (Uxio0) [inline manage.py]: I will remove this, as on production we inject the environments from kubernetes and I bet this will conflict

**Review by Uxio0 — APPROVED** [review]

**Pull request merged**